lua/entities/ent_jack_turret_base/init.lua

```diff
@@ -51,6 +51,7 @@ ENT.RoundsOnBelt = 0
 ENT.NextWarnTime = 0
 ENT.WillWarn = false
 ENT.WillLight = false
+ENT.WillLightOverride = nil

```

Comment:
Is this overriding a not-nil value? If not, you can remove this line

Reply:
I put it there so it's with the other configurable variables.